supportFiles/playerStats.js

```diff
@@ -0,0 +1,170 @@
+            break;
+        case 'Ithorian':
+            // homePlanet = 'Ithor';
+            homePlanet = mapBuild.galaxyMap[2][0];
+            break;
+        case 'Klatooinians':
+            // homePlanet = 'Klatooine';
+            homePlanet = mapBuild.galaxyMap[2][2];
+            break;
+        case 'Mon Calamari':
+            // homePlanet = 'Mon Cala';
+            homePlanet = mapBuild.galaxyMap[3][1];
+            break;
+        case 'Neimoidian':
+            // homePlanet = 'Neimoidia';
+            homePlanet = mapBuild.galaxyMap[3][4];
+            break;
+        case 'Quarren':
+            // homePlanet = 'Mon Cala';
+            homePlanet = mapBuild.galaxyMap[3][1];
+            break;
+        case 'Rodian':
+            // homePlanet = 'Rodia';
+            homePlanet = mapBuild.galaxyMap[4][1];
+            break;
+        case 'Sullustan':
+            // homePlanet = 'Sullust';
+            homePlanet = mapBuild.galaxyMap[5][0];
+            break;
+        case 'Togruta':
+            // homePlanet = 'Shili';
+            homePlanet = mapBuild.galaxyMap[4][4];
+            break;
+        case 'Trandoshan':
+            // homePlanet = 'Trandosha';
+            homePlanet = mapBuild.galaxyMap[5][4];
+            break;
+        case 'Twi\'lek':
+            // homePlanet = 'Ryloth';
+            homePlanet = mapBuild.galaxyMap[4][3];
+            break;
+        case 'Ugnaught':
+            // homePlanet = 'Gentes';
+            homePlanet = mapBuild.galaxyMap[1][5];
+            break;
+        case 'Weequay':
+            // homePlanet = 'Sriluur';
+            homePlanet = mapBuild.galaxyMap[4][5];
+            break;
+        case 'Zabrak':
+            // homePlanet = 'Iridonia';
+            homePlanet = mapBuild.galaxyMap[2][1];
+            break;
+        default:
+            console.clear();
+            console.log('You need to choice a species');
+            playerSpecies();
+            // homePlanet = 'ID-10T';
+    }
+    return selectedSpecies;
+}
+const classSelection = () => {
+    classList = [];
+    classList[0] = 'Beast Master';
+    classList[1] = 'Bounty Hunter';
+    classList[2] = 'Chef';
+    classList[3] = 'Commando';
+    classList[4] = 'Craftsman';
+    classList[5] = 'Entertainer';
+    classList[6] = 'Jedi';
+    classList[7] = 'Medic';
+    classList[8] = 'Officer';
+    classList[9] = 'Politician';
+    classList[10] = 'Ranger';
+    classList[11] = 'Smuggler';
+    classList[12] = 'Spy';
+    classList[13] = 'Trader';
+    selectedClass = readline.keyInSelect(classList, 'Select your starting class', {cancel: 'Please choose a class from above'});
+
+    if (selectedClass !== -1) {
+        return selectedClass;
+    } else {
+        console.clear();
+        console.log('You need to choice a class');
+        classSelection();
+    }
+}
+const getCharacter = () => {
+    return playerCharacter;
+}
+
+// createCharacter();
+// console.log(playerCharacter.class)
+
```

Comment:
Great job on this game, and really awesome to see you importing and exporting functions that will help. Fantastic job!